### PR TITLE
Update client to match latest server and assets.json format changes

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,60 +23,69 @@ function readAssetsJson(path) {
...
             );
         }
-        if (inputs.css) {
+        if (cssInput) {
             this.styles.push(
                 new AssetCss({
-                    ...tagOptions.css,
-                    value: `${server}/${organisation}/css/${name}/${version}/index.css`,
+                    ...cssOptions,
+                    value: `${server}/${organisation}/pkg/${name}/${version}/main/index.css`
```
Comment (trygve-lie) [inline index.js]: In core I've made some helper functions to construct these, should they go into a common module for reuse perhaps?

File: index.js [hunk 2]
```diff
@@ -29,60 +23,69 @@ function readAssetsJson(path) {
...
 
-        if (inputs.js) {
+        if (jsInput) {
             this.scripts.push(
                 new AssetJs({
-                    ...tagOptions.js,
-                    value: `${server}/${organisation}/js/${name}/${version}/index.js`,
                     type: 'module',
+                    ...jsOptions,
+                    value: `${server}/${organisation}/pkg/${name}/${version}/main/index.js`
```
Comment (trygve-lie) [inline index.js]: I would use the node `path` module to join these. Then you do not need to think about `/` becoming correct.

**Review by trygve-lie — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (digitalsadhu) [reply index.js]: was wondering about that, worth it I think for sure

File: index.js (hunk 2 above)
Comment (digitalsadhu) [reply index.js]: good point

**Pull request merged**